README.rst

```diff
@@ -340,6 +340,38 @@ function:
        pytest.fail("+++ Timeout +++")
 
 
+
+Session Timeout
+===============
+
+The above mentioned timeouts are all per test function. You can also set a
+session timeout in seconds. The following example shows a session timeout
+of 10 minutes (600 seconds)::
+
+   pytest --session-timeout=600
+
+You can also set the session timeout the pytest configuration file using the ``session_timeout`` option:
+
+   .. code:: ini
+
+      [pytest]
+      session_timeout = 600
+
+Friendly timeouts
```

Comment:
maybe "cooperative timeouts" is a better title for this?  I feel like this is a concept (cooperative multi-tasking) that maps pretty well to how this behaves.  Still, as above I would first describe both how it behaves and why it is called as such.

Reply:
sounds good